protocol/message.go

```diff
@@ -0,0 +1,50 @@
+package protocol
+
+import "github.com/coniks-sys/coniks-go/merkletree"
+
+// Defines constants representing the types
+// of messages exchanged by clients and servers.
+const (
+	RegistrationType = 0
+	CommitmentType   = 1
```

Comment:
Ah ok. I actually need to change this in the Java implementation since commitments is an old term we used for STRs. But I don't think there's a separate STR message in this implementation, at least for now, right? So I think you could remove it and make `LookupType = 1`so you don't have the awkwardness.